edge/pkg/devicetwin/dtmanager/dmiserver.go

```diff
@@ -0,0 +1,427 @@
+/*
```

Comment:
the file name dmiserver is strange, we have two dmiserver, it is should be dmiworker.go

Reply:
done